src/ws.nim

```diff
@@ -147,6 +150,19 @@ type
 template remainder*(frame: Frame): uint64 =
   frame.length - frame.consumed
 
+proc toString*(ht: HttpTables): string =
```

Comment:
Usually this is accomplished with the dollar (`$`) operator,  you can even leave the `toString` proc and just call it from the `$` proc.

```suggestion
proc toString*(ht: HttpTables): string =
...

proc `$`*(ht: HttpTables): string = ht.toString
```

When you want to convert to string you simply prepend the variable with the `$`, like so

```nim
echo $ht
```

Reply:
Done